flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java

```diff
@@ -76,6 +77,8 @@
 
 	public static final String TABLE_SCHEMA_TYPE = "type";
 
+	public static final String TABLE_SCHEMA_EXPR = "expr";
```

Comment:
`TABLE_SCHEMA_EXPR` looks obscure to me.

Reply:
Thanks, do you have a better name ?

Reply:
`TABLE_SCHEMA_COMPUTED_COLUMNS`?

Reply:
I'm inclined to use `TABLE_SCHEMA_EXPR` also rename `watermark.strategy.expression` to `watermark.strategy.expr`

Reply:
To be honest, I prefer `datatype` than `type`. The community had some discussions about the misusing of `type`, e.g. `connector.type`, `format.type`, `schema.type`, it is confused what's the `type` meaning.

Reply:
Hi @twalthr , do you have some ideas on this?